set.mm

```diff
@@ -71054,6 +71061,28 @@ Axiom of Replacement (unlike ~ resfunexg ).  (Contributed by NM,
     PKEZUSULUTUMULUOABACUAZPQULUMCASZKEZVAULCUIUMVDACUBCABUCUDULVAVDULUPVCKULUP
     CUOSVCCUEULUOACVBUFUGPQUHUOUPBKUJTCUQKUKT $.
 
+  $( Weak version of ~ funex that holds without ~ ax-rep .  If the domain and
+     codomain of a function exist, so does the function.  (Contributed by Rohan
+     Ridenour, 13-Aug-2023.) $)
+  funexw $p |- ( ( Fun F /\ dom F e. B /\ ran F e. C ) -> F e. _V ) $=
+    ( wfun cdm wcel crn w3a cxp cvv xpexg 3adant1 wss wrel funrel relssdmrn syl
+    3ad2ant1 ssexd ) CDZCEZAFZCGZBFZHCUAUCIZJZUBUDUEUFFTUAUCABKLTUBCUEMZUDTCNUG
+    COCPQRS $.
+
+  ${
+    $d x A $.  $d x C $.
+    mptexw.1 $e |- A e. _V $.
+    mptexw.2 $e |- C e. _V $.
+    mptexw.3 $e |- A. x e. A B e. C $.
+    $( Weak version of ~ mptex that holds without ~ ax-rep .  If the domain and
+       codomain of a function given by maps-to notation are sets, the function
+       is a set.  (Contributed by Rohan Ridenour, 13-Aug-2023.) $)
+    mptexw $p |- ( x e. A |-> B ) e. _V $=
```

Comment:
@avekens : wouldn't we prefer to have a weak version of mptexg instead ?

Reply:
Yes, a weak version mptexgw of mptexg would be preferred, and mptexw could be derived from it easily (in the same way as mptex is derived from mptexg).

Reply:
I'm going to merge. Theorem mptexw could be replaced with mptexgw in a later PR (but you don't have to).